### DISCUSSION TITLE
failed to modify open files, cfs-server start failed if no root user to start

### DISCUSSION THREAD

**Discussion by Victor1319** [cubefs/cubefs#2368] [Q&A] [answered]
`err(readFromProcess: sub-process: [cmd.go 256] Fatal: failed to modify open files - Error Setting Rlimit operation not permitted)
`

**Comment by Victor1319** [comment, marked as answer]
Please check `cat /etc/security/limits.conf`, to look up the limit config
It should be bigger than the num config `1024000` or not limit.
Because the user, not root, can't modify the number over limit int `limits.conf`

**Reply by xiaojunxiang2023** [reply]
建议谨慎修改（有些服务器不支持 1024000这么大），我刚强行修改后连 ssh都登不上去了。还好有另一个终端没退，回退后 ssh才可以继续用。cubef把 1024000 做成硬编码了不可配置，这个我感觉很不好。
